pyclesperanto_prototype/_tier9/_statistics_of_labelled_pixels.py

```diff
@@ -1,32 +1,46 @@
+from warnings import warn
+
 from skimage.measure import regionprops
 
 from .._tier0 import Image
 from .._tier0 import create_none
 from .._tier0 import plugin_function
 
 @plugin_function(output_creator=create_none)
-def statistics_of_labelled_pixels(input : Image = None, labelmap : Image = None, measure_shape : bool = True, extra_properties = []):
+def statistics_of_labelled_pixels(input : Image = None, labelmap : Image = None, measure_shape : bool = True, extra_properties = [], use_gpu : bool = True):
```

Comment:
See above. Note also that in skimage we allow `input` to be None, since many region properties can be measured without the image intensities. Also note that `input()` is a Python built-in function, so it's best to avoid its use as a keyword argument. (Yes, `scipy.ndimage` uses it, but this has led to multiple debates over the years about deprecating it.)